### DISCUSSION TITLE
3.3.2?

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3679] [Team Posts]
@jekkos when are we going to go live with 3.3.2?  I'm sorry to push, but I've got a couple of pull requests that we are waiting on merging until after 3.3.2 and my backlog is piling up.

**Comment by jekkos** [comment]
Ok sorry for the delay again. I'll press the release button now, we have waited long enough. Let's fix any issues after.

**Comment by objecttothis** [comment]
Super.  Thanks.  Now I just need to see if anyone is available to do a code review on #2821

**Comment by jekkos** [comment]
I will be bikepacking coming week, so will see if I find some time in between to have a closer look.

**Comment by objecttothis** [comment]
> I will be bikepacking coming week, so will see if I find some time in between to have a closer look.

Thank you @jekkos.  Enjoy the trip!

**Comment by jekkos** [comment]
OK great, I have pushed the button . I'll be leaving on sunday but will be busy doing other stuff. The build did deliver a nicely tagged 3.3.2 on bintray that I now have attached to the release itself.
